rest-api/src/main/scala/za/co/absa/enceladus/rest_api/controllers/v3/VersionedModelControllerV3.scala

```diff
@@ -87,6 +89,12 @@ abstract class VersionedModelControllerV3[C <: VersionedModel with Product
     forVersionExpression(name, version) { case (name, versionInt) => versionedModelService.getUsedIn(name, Some(versionInt)) }
   }
 
+  @GetMapping(Array("/{name}/used-in"))
+  @ResponseStatus(HttpStatus.OK)
+  def usedIn(@PathVariable name: String): CompletableFuture[UsedIn] = {
```

Comment:
No, those should be probably `@JsonIgnore`d. Thanks for noticing and done.

Btw. On that account, I have also noticed that should there be any actual existing `MenasReference`s, their `collection` is always empty - I have searched this to be the case since 2019 when it appeared and it is also recorded exactly like this in the integration tests. I wonder, however, why is this so...?